db/academyPortal_password.sql

```diff
@@ -0,0 +1,78 @@
+# ************************************************************
```

Comment:
please delete this file. there should be **one** database file, named the same as the original